tests/unit/archive_base_test.py

```diff
@@ -405,6 +405,407 @@ def test_check_missing_files(datapackage, baseline_resources, new_resources, suc
     assert validation_result.success == success
 
 
+@pytest.mark.parametrize(
+    "baseline_resources,new_resources,success",
+    [
+        (
+            # Test when file size changes by > threshold for any one file
```

Comment:
non-blocking: You can name these test cases with `pytest.param(value, other_value, id="some name")` which will make test output a little bit nicer.

Reply:
Done, great suggestion